linter/lints/rfc/lint_crl_has_issuer_name.go

```diff
@@ -0,0 +1,64 @@
+package rfc
+
+/*
+ * ZLint Copyright 2023 Regents of the University of Michigan
```

Comment:
How come this gained a copyright notice for the Regents of the University of Michigan?

If we're planning to contribute this lint upstream, wouldn't we add the copyright notice at the time we contribute it (assuming upstream has a relevant requirement about copyright notices)?

Reply:
These were accidental, they get removed in the last commit in the stack.